wiki/News_styling_criteria/en.md

```diff
@@ -68,61 +68,34 @@ date: 2017-08-17 03:00:00 +0000
-
-#### Headers
-
-Header levels 1 (``#``), 2 (``##``), and 3 (``###``) can be used. They have unique identifiers and can be linked directly to. Never use headers to style or format text.
-
-#### Bold
-
-Bolding is reserved for highlighting objects or phrases of importance, be them players, scoring, or whatever else is contextually considered a focus of the article in question.
-
-#### Italics
-
-Italics is used uncommonly to accent phrases or statements of interest within the conversational register. It may also be used to designate names of other creative works, such as the *osu! Coffee Hour* or the *Aspire Beatmapping Contests*.
-
-#### Horizontal bar
-
-Horizontal bars must be used sparingly and only in situations where content needs to be cleanly and definitively separated from the rest of the article.
-
-#### Tables
-
-Markdown tables can be used, but is not recommended for displaying data in news articles. Visualising the data using an image is likely to be more pleasing and more accessible to most users.
-
-### Images
-
-All images linked to news articles must either be hosted on osu! servers (i.e. `assets.ppy.sh`) or placed within and referenced from the `osu-wiki` shared image repository.
-
-News articles that use images have their own folder within ``/wiki/shared/news/`` using the same naming format as their filename, with the exception of the ``.md`` suffix.
-
-Example:
-
-``/wiki/shared/news/2017-08-17-news-article-standards/banner.jpg``
-
-#### Covers
+- Titles
+- Headers
+  - Header levels 1 (``#``), 2 (``##``), and 3 (``###``) can be used. They have unique identifiers and can be linked directly to. Never use headers to style or format text.
+- Bold
+- Italics
+- Horizontal bar
+- Tables
+- Images
+  - Images linked to news articles must either be hosted on osu! servers (i.e. `assets.ppy.sh`) or placed within and referenced from the `osu-wiki` shared image repository. News articles that use images have their own folder within ``/wiki/shared/news/`` using the same naming format as their filename, except for the ``.md`` suffix. Example:
+  ``/wiki/shared/news/2017-08-17-news-article-standards/banner.jpg``
+- Covers
+  - Cover images should have a minimum width of 900 and minimum height of 200 for their base size. A `@2x` version with each dimension doubled should be provided if the source image is large enough to support it. Cover images appearing in multiple news articles are commonly put in the `/wiki/shared/news/banners` directory. <!-- TODO: there should be a more general place to explain how and why @2x images can be used, this isn't enough info and it's not only for cover images. -->
 
 News articles should have a [block image](/wiki/Article_styling_criteria/Formatting#display) after the preview paragraph for use as a **cover** (sometimes called *banner*). The game client and website automatically use the first image in an article as a cover on the front page, news listing, and news article view.
```

Comment:
relevant info:

- this banner should never have any alt text (the text in the square brackets of an image link)
- the paragraph immediately above the banner image is used as the preview

```suggestion
News articles should have a [block image](/wiki/Article_styling_criteria/Formatting#display) after the preview paragraph for use as a **banner** (a.k.a. *cover*). The game client and website automatically use the first image in an article as a banner on the front page, news listing, and news article view.
```

Reply:
let me know if the latest push is ok with you